modules/margin/src/main/java/com/opengamma/sdk/margin/MarginWhatIfCalcResult.java

```diff
@@ -59,17 +59,79 @@
    */
   @PropertyDefinition(validate = "notNull")
   private final List<PortfolioItemSummary> portfolioItems;
+
+  /**
+   * The result of the margin calculation for the base portfolio
+   */
+  @PropertyDefinition
+  private final MarginSummary baseSummary;
+
+  /**
+   * The result of the margin calculation for the combined portfolio (base + delta)
+   */
+  @PropertyDefinition
+  private final MarginSummary combinedSummary;
+
   /**
-   * The result of the margin calculation
+   * The result of the margin calculation for the delta portfolio
    */
-  @PropertyDefinition(get = "optional")
-  private final MarginWhatIfSummary margin;
+  @PropertyDefinition
+  private final MarginSummary deltaSummary;
+
   /**
    * The list of failures that occurred, may be empty.
    */
   @PropertyDefinition(validate = "notNull")
   private final List<MarginError> failures;
 
+  /**
+   * Creates a new instance of {@link MarginWhatIfCalcResult} with the given details.
+   *
+   * @param status The request status, as an instance of {@link MarginCalcResultStatus}.
```

Comment:
preference is for "The" to be "the" and no full stop - ie, a phrase not a sentence